widgetssdk/src/main/res/layout/entry_widget_powered_by_item.xml

```diff
@@ -12,12 +13,13 @@
         android:layout_marginEnd="@dimen/glia_small"
         android:layout_marginBottom="@dimen/glia_medium"
         style="@style/Application.Glia.Caption"
-        android:textColor="?attr/gliaBaseShadeColor"
+        android:textColor="@color/glia_branding_color"
```

Comment:
will this be properly handled if the integrator has a `gliaBaseShadeColor` declaration in their theme?